### ISSUE TITLE
Missing source map files for ui-box package dependency

### ISSUE THREAD

**Issue by carmouche** [segmentio/evergreen#761]
I wasn't sure whether to put this into the issues for ui-box or for evergreen, but I'm using this package specifically so feel free to move it if it doesn't belong :) 

## Issue
I'm using Parcel to bundle my code and am running into a warning related to missing files in source maps for ui-box.  There's sadly no way to exclude source maps for specific packages, and the issues I looked at say it's related to the published package.

I'm using `"evergreen-ui": "^4.23.3"` 
Here is a screenshot of the warnings for the missing files:
![image](https://user-images.githubusercontent.com/13294714/77214528-d523fe80-6acc-11ea-87cc-667858666ea1.png)

And here are some related issues I found that suggest it is related to how the package is being published.  Specifically: "it's just a warning saying the source maps are referring to files that aren't there."
https://github.com/parcel-bundler/parcel/issues/2356
https://github.com/parcel-bundler/parcel/issues/2346

I'm happy to provide more details or assist in fixing the issue if needed!  Thanks!

**Comment by mshwery** [comment]
This seems like an issue with `ui-box` not evergreen. Can we move the issue there?

**Comment by mshwery** [comment]
From what it looks like this isn't a failure in parcel – this is parcel warning you that these sourcemaps are missing from a package. IMO it's weird that parcel expects all packages to have sourcemaps.

**Comment by carmouche** [comment]
Hey!  Thanks so much for the quick reply!
I think if I read correctly in the issues I saw, the error indicates that there are source maps that are generated, but the warnings are for source maps that point to files that do not exist. 

Maybe it's just a file cleanup or build-related issue?  I agree it would be unreasonable to require source maps if that is the case, but I don't think so.  

Btw to clarify, when I said "There's sadly no way to exclude source maps for specific packages", I meant I can't exclude them per-package if they exist, but I don't think they are required. My build runs pretty well and certainly, I'm not blocked, but my buddy at work mentioned that there were Parcel warnings that caused our CI build to be slower, so I investigated :)
